db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: sqitch; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA sqitch;
```

Comment:
One reason why I didn't create individual migrations for the plugins is that I wasn't sure how this would all work.  Like, if you tried to apply this migration on one of our servers, the ``sqitch`` schema already exists, because it was created in our core ``uber`` plugin migrations, right?

Another reason is that I wasn't clear on how these migrations would work if you ran into situations like we had here:
- ``uber`` defines a set of migrations which create a set of columns on the ``Attendee`` table
- ``hotel`` defines a migration which adds columns to the ``Attendee`` table
- ``uber`` defines further migrations which also modify the ``Attendee`` table
- more migrations happen

Are migrations expected to be sequential?  In some migration frameworks, there's a set of migration ids which are supposed to happen in order, but there's no canonical ordering of the above.  That's why I was thinking that instead of putting this kind of thing in the individual feature plugins like ``hotel`` and ``panels``, we'd put it in the event-specific plugins like ``magclassic`` or ``magprime``.  And then those events would use their own migrations which include ALL of the plugins they care about, instead of having each feature plugin track its own migrations in ways which might not interact well.

On the other hand, if you've tested this out and are confident that it's a feasible approach, we can try it this way.